bin/base.sh

```diff
@@ -16,3 +16,13 @@ if ( command -v pigz >/dev/null 2>&1 ); then
 else
     export GZIP_CMD=gzip
 fi
+
+if ( command -v pv >/dev/null 2>&1 ); then
+    PV() {
+        pv "$@"
+    }
+else
+   PV() {
+       cat
```

Comment:
Can you add a comment her saying that the arguments are being ignored and why?

Reply:
Done